.github/workflows/documentation.yml

```
@@ -34,7 +34,7 @@ jobs:
           sudo apt-get update
           sudo apt-get install libsqlite3-dev
           sudo apt-get install libsodium-dev
-          sudo apt-get install g++-9
+          sudo apt-get install g++-9 clang
```

```suggestion
          sudo apt-get install clang
```

Don't think we need `g++-9` anymore, do we ? LDC might still use it for linking, but that's it.